internal/kube/site/site.go

```diff
@@ -1163,9 +1163,12 @@ func (s *Site) updateRouterAccessStatus(la *skupperv2alpha1.RouterAccess) {
 }
 
 func asSecuredAccessSpec(la *skupperv2alpha1.RouterAccess, group string, defaultIssuer string) skupperv2alpha1.SecuredAccessSpec {
```

Comment:
```suggestion
func asSecuredAccessSpec(routerAccess *skupperv2alpha1.RouterAccess, group string, defaultIssuer string) skupperv2alpha1.SecuredAccessSpec {
```

(just so the parameter is more descriptive)